changelogs/unreleased/3945-docs-relation-precedence-policy.yml

```diff
@@ -0,0 +1,7 @@
+---
+description: Add documentation regarding the relation precedence policy feature to the troubleshooting documentation
+issue-nr: 3945
+change-type: minor
+destination-branches: [master, iso5]
+sections:
+  feature: "{{description}}"
```

Comment:
Does this require a change entry? The feature itself was just added so the documentation would be assumed to be part of that.